telemetry/definitions/commonDefinitions.json

```diff
@@ -2796,6 +2796,14 @@
                 {
                     "type": "codeTransformSessionId",
                     "required": true
+                },
+                {
+                    "type": "reason",
+                    "required": false
+                },
+                {
+                    "type": "result",
+                    "required": false
                 }
```

Comment:
these are standard on all metrics. do we need to update the Kotlin generator so that it reflects that?